### PR TITLE
Deploy to protosaur.dev (fixes #60)

### LINKED ISSUE #60: Deploy a copy of Glean Dictionary to protosaur.dev
*This bug requires specialized knowledge and access to Mozilla's internal systems, so is not a good issue for contributors*

We should deploy a copy of the Glean dictionary to protosaur.dev on a regular basis. protosaur currently requires auth (making it inaccessible to those outside Mozilla), but that should be fixed soon by mozilla/protodash#16

### PR DESCRIPTION
**Pull request by wlach**
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,8 +29,38 @@ jobs:
...
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - attach_workspace:
+          at: /tmp/workspace-data
+      - run: cp -r /tmp/workspace-data/* /tmp/workspace/public/data
+      - gcs/gcs-rsync:
+          bucket: glean-dictionary-dev
+          source: /tmp/workspace/public
+          options: -r
```
Comment (acmiyaguchi) [inline .circleci/config.yml:54]: Is `-d` appropriate here when files are removed?

File: .circleci/config.yml [hunk 2]
```diff
@@ -22,8 +29,38 @@ jobs:
...
+    docker:
+      - image: google/cloud-sdk
+    working_directory: /tmp
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - attach_workspace:
+          at: /tmp/workspace-data
+      - run: cp -r /tmp/workspace-data/* /tmp/workspace/public/data
+      - gcs/gcs-rsync:
```
Comment (acmiyaguchi) [inline .circleci/config.yml:51]: How did you provision the service account for this?

**Review by acmiyaguchi — APPROVED** [review]
Looks good to me, with comments addressed.

File: .circleci/config.yml (hunk 2 above)
Comment (acmiyaguchi) [reply .circleci/config.yml:51]: I'd recommend the importing the service account into https://github.com/mozilla-services/data-sandbox-terraform/blob/master/projects/glean-dictionary/project/main.tf if this was created in the GCP console.

File: .circleci/config.yml (hunk 1 above)
Comment (wlach) [reply .circleci/config.yml:54]: I think so, can add this.

File: .circleci/config.yml (hunk 2 above)
Comment (wlach) [reply .circleci/config.yml:51]: I'll file a PR

**Pull request merged**